CoreImportExportPlugins/src/au/gov/asd/tac/constellation/plugins/importexport/RowFilter.java

```diff
@@ -93,9 +94,25 @@ public boolean setScript(final String script) {
             return false;
         }
     }
+    private String modifyScript(final String script) {
+        final int endOfHeader = script.indexOf("==");
+        String modifiedScript = script;
+        if (endOfHeader != -1) {
+            final String header = script.substring(0, endOfHeader);
+            if (header != null && !header.matches("^[a-zA-Z][a-zA-Z0-9_]*$")) {
+                //Append the script with the relevant ith column header containing columnHeaderPrefix
```

Comment:
Did you mean to say "Amend" instead?

Reply:
I don't think append is necessarily the wrong choice of word here but "Edit" may be the better choice (I usually associate append with adding to the beginning or end rather than somewhere in the middle)